core/src/components/progress-bar/test/basic/progress-bar.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('progress-bar: basic', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto('/src/components/progress-bar/test/basic');
+
+    await page.setIonViewport();
+
+    expect(await page.screenshot()).toMatchSnapshot(`progress-bar-basic-${page.getSnapshotSettings()}.png`);
```

Comment:
We need to disable animations here otherwise the screenshot will be flaky due to the first few progress bars.

See the "animations" option here: https://playwright.dev/docs/test-assertions#locator-assertions-to-have-screenshot-2

(same thing below)

Reply:
Updated with that change 👍 oddly enough the new capture matches the same screenshot diff (at least from pixelmatch determines a diff). So you won't see a new commit from Ionitron for adding new screenshots after the change.

Reply:
Doesn't our custom Playwright setup automatically disable animations with `?ionic:_testing=true`?

Reply:
Oh right, I forgot that the progress bar integrates with that. I had originally through it only disabled animations created via JS: https://github.com/ionic-team/ionic-framework/blob/b6422432149cb12ec90aa09b051761bf0c06007e/core/src/components/progress-bar/progress-bar.tsx#L60

Feel free to remove https://github.com/ionic-team/ionic-framework/pull/25807/commits/6a606a1151f5b89ac9fd8b680a25858d834d7638 then.